scripts/install_and_disable_systemd

```diff
@@ -0,0 +1,125 @@
+#! /usr/bin/env bash
+
+function install_and_disable_systemd
+{
+    if [ -z "$1" ]
+    then
+        # shellcheck disable=SC1078,SC1079
+        echo """Usage: source install_and_disable_systemd PATH [NAMES]
+
+    Installs all services from the paths in PATH(colon seperated) and disables and removes
+    all services which matches the names in NAMES (colon seperated).
+
+    The sevices to be disabled are disable before the new ones are installed.
+    """
```

Comment:
indents

Reply:
The indent of the text in the comment is correct. There needs to be this indent, so that it is printed correctly.